packages/input-text/package.json

```diff
@@ -36,6 +30,6 @@
   },
   "dependencies": {
     "@crave/farmblocks-hoc-disabled-tooltip": "^1.0.1",
-    "@crave/farmblocks-hoc-input": "^1.1.1"
+    "@crave/farmblocks-hoc-input": "^1.2.0-alpha.4b468a5d"
```

Comment:
lets change this back and trust that the semantic release will do the right thing :)

Reply:
I removed the commit